translator/translate/otel/pipeline/jmx/translator.go

```diff
@@ -0,0 +1,105 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package jmx
+
+import (
+	"fmt"
+	"strconv"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/confmap"
+
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/common"
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/exporter/awscloudwatch"
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/extension/agenthealth"
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/processor/ec2taggerprocessor"
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/processor/filterprocessor"
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/processor/resourceprocessor"
+	"github.com/aws/amazon-cloudwatch-agent/translator/translate/otel/receiver/jmx"
+)
+
+const (
+	placeholderTarget = "<type>"
```

Comment:
Not sure what the placeholder should be in the error. `<type>` or `<target>` or `<target-system>`.

Reply:
Had some discussions and it seems like we're going to go with `<target-system>` to match the OTEL terminology.